README.md

```diff
@@ -20,6 +20,13 @@ Join us to help define the direction and implementation of this project!
 - Use [GitHub Issues](https://github.com/open-policy-agent/gatekeeper/issues)
   to file bugs, request features, or ask questions asynchronously.
 
+## How is Gatekeeper different from OPA?
+Gatekeeper extends the [OPA with its sidecar kube-mgmt](https://www.openpolicyagent.org/docs/kubernetes-admission-control.html) (aka Gatekeeper v1.0) by introducing the following functionality:
```

Comment:
"extends" may suggest we're building on top of OPA + kube-mgmt, which isn't accurate.

Maybe rephrase to "Compared to using [OPA with the kube-mgmt sidecar], Gatekeeper introducess the following functionality:"

At a minimum drop the "the" from "extends the" (for grammar's sake).

Do we want to mention that Config Maps are not a valid way to configure Gatekeeper? Moving away from those was a request at Kubecon Seattle.